reactor-netty-http/src/main/java/reactor/netty/http/client/HttpClientConnect.java

```diff
@@ -265,7 +264,7 @@ public void subscribe(CoreSubscriber<? super Connection> actual) {
 
 				AddressResolverGroup<?> resolver = _config.resolverInternal();
 
-				_config.connectionProvider()
+				_config.httpConnectionProvider()
```

Comment:
I don't know if it's best to use the implementation directly without using the interface.

Reply:
It is ok, one and the same implementation is used before and now.